src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const newDeclaration = declaration.trim();
+
+    if (!declaration) {
+      return;
```

Comment:
The condition in the if statement is checking the wrong variable. It should check 'newDeclaration' instead of 'declaration' since 'newDeclaration' is the trimmed version of the declaration string.